### PR TITLE
Manu changes happened since the last update

### PR DESCRIPTION
**Pull request by sparkybg**

### CODE REVIEW

**Comment by valerionew** [comment]
Hey! I hope you're doing well. 
What happend with the two transistors that were removed?

**Comment by sparkybg** [comment]
Hi,

I'm well. Hope you are too. :)

If you are asking about Q15 and Q20:
- Q15 was meant to drive the middle indicator when 7-segment indicators are used, which never happened.
- Q20 was meant to drive a buzzer with internal generator, which again never happened.

The rest is more or less the same. Only 4-layer boards and separate wiring for display power is added.

...and, it now supports partial power, which is very important for small instruments like C210, and C105/C115.

**Comment by sparkybg** [comment]
By the way, I don't know how to rebase to latest. Any help?

**Comment by valerionew** [comment]
I was thinking of the edit you did on the readme that removed this paragraph:
https://github.com/valerionew/unisolder-notes/pull/32/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L262-L264

> #### Q10, Q11: IPD053N08
> If you want to change this part, pick a part which has same: package, Rds(on), Vds, and gate threshold voltage.

Also: no problem for the rebase, it'll rebase automaticall when merging

**Comment by sparkybg** [comment]
They are there. They were mentioned 2 times in the readme. Some rows below there is:

Q10, Q11 IPD053N08
AUIRFR3607 or IPD12CN10NGATMA1 can do the job.

To substitute this part the important parameters are:

Vds(max) should be at least 75V.
Rds(on) should be no more than 0.01ohm.

So, the first one was not needed and I removed it.

**Comment by valerionew** [comment]
You are right! Sorry about that. Merging

**Review by valerionew — COMMENTED** [review]
Solved, clarified in #32

File: README.md [hunk 1]
```diff
@@ -259,9 +262,6 @@ Follows a table with values that should suit. These values are not tested (FAN53
 
 ### Transistors
 
-#### Q10, Q11: IPD053N08
-If you want to change this part, pick a part which has same: package, Rds(on), Vds, and gate threshold voltage.
-
```
[the remark this answers is not in the document]
Comment (sparkybg) [reply README.md:264]: Everything is OK. It is mentioned that 75V MOSFET must be used, with same or lower RDSon. This is the only important parameters.

**Comment by sparkybg** [comment]
Please also rebase the sourse repo. It still points to older one.

**Pull request merged**